skema/gromet/execution_engine/query_runner.py

```diff
@@ -1,19 +1,23 @@
 import yaml
 import traceback
 from pathlib import Path
-from gqlalchemy import Memgraph
+
+from neo4j import GraphDatabase
```

Comment:
Ack!  I wasn't thorough enough in my review.  It looks like we haven't added this to `pyproject.toml`.  We may also want to remove the old dep we're replacing here.

Reply:
Ah you are right, let me fix that.